src/__tests__/applicationForms.int.test.ts

```diff
@@ -202,24 +202,28 @@ describe('/applicationForms', () => {
 				baseFieldShortCode: 'yearsOfWork',
 				position: 1,
 				label: 'Anni Worki',
+				instructions: 'Please enter the number of years of work.',
 			});
 			await createApplicationFormField(db, null, {
 				applicationFormId: 3,
 				baseFieldShortCode: 'organizationName',
 				position: 2,
 				label: 'Org Nomen',
+				instructions: 'Please enter the name of the organization.',
```

Comment:
This is moot since you did it already, but you should feel OK with just making this null.  It's more test boilerplate and null is just as valid a value as a string.

Since you've written the tests with values already no need to spend time removing them!

That said, make sure to create a test that explicitly checks to confirm that `null` is a settable value for creation.